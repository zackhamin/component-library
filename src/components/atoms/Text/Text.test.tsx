import { Text } from '.';
import { render, screen } from '@testing-library/react';

describe('Text', () => {
	it('renders correctly', () => {
		const { container } = render(<Text>Text</Text>);
		expect(container).toMatchSnapshot();
	});

	it('renders as a span', () => {
		const { container } = render(<Text as="span">Text</Text>);

		const span = container.querySelector('span');

		expect(span).toBeInTheDocument();
	});
});
