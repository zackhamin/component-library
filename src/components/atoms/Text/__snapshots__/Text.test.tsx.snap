// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text renders correctly 1`] = `
.c0 {
  margin: 0;
  font-weight: 400;
  font-family: Helvetica,Arial,sans-serif;
  color: #333333;
  font-size: 1.6rem;
  line-height: 160%;
}

<div>
  <p
    class="c0"
  >
    Text
  </p>
</div>
`;
